### PR TITLE
Fetch event info

### PR DESCRIPTION
**Pull request by kathlynnguyenh11**

### CODE REVIEW

File: src/components/ContactOption.jsx [hunk 1]
```diff
@@ -10,12 +10,15 @@ function ContactOption(props) {
...
-  const [defaultValue, setDefaultValue] = useState(contacts[0].name);
+  const [defaultValue, setDefaultValue] = useState(null);
 
   const fetchContacts = () => {
     fetch(BASE_URL, { method: 'GET' })
       .then((response) => response.json())
-      .then((data) => setContacts(data));
+      .then((data) => {
+        setContacts(data);
+        setDefaultValue(data[0].id);
```
Comment (peppermintpatty5) [inline src/components/ContactOption.jsx]: Assumes that `data` is non-empty

Comment (kathlynnguyenh11) [reply src/components/ContactOption.jsx]: I'll update it to make it work for empty data

Comment (kathlynnguyenh11) [reply src/components/ContactOption.jsx]: Just updated it now

File: src/components/ContactOption.jsx [hunk 2]
```diff
@@ -17,7 +17,9 @@ function ContactOption(props) {
       .then((response) => response.json())
       .then((data) => {
         setContacts(data);
-        setDefaultValue(data[0].id);
+        if (data !== null) {
```
Comment (peppermintpatty5) [inline src/components/ContactOption.jsx]: You should be checking
```js
if (data.length !== 0)
```

Comment (kathlynnguyenh11) [reply src/components/ContactOption.jsx]: This makes more sense. Thanks! I updated

**Review by peppermintpatty5 — APPROVED** [review]

**Pull request merged**